packages/design-system/src/utils/prepareFrameStatsComponent.ts

```diff
@@ -77,7 +77,10 @@ export default function prepareFrameStatsComponent(
   return {
     dataMapping: [
       {
-        title: 'Frame details',
+        title:
```

Comment:
@fellyph 

As I mentioned, we need to change the "Frame details" to **"Frames"** and not "Frames details"

Reply:
I have changed now